WooCommerce/Classes/POS/Utils/AdaptiveButtonPaddingModifier.swift

```diff
@@ -0,0 +1,33 @@
+import SwiftUI
+
+struct AdaptiveButtonPaddingModifier: ViewModifier {
+    @Environment(\.dynamicTypeSize) private var dynamicTypeSize
+    let defaultButtonPadding: CGFloat
+
+    func body(content: Content) -> some View {
+        content
+            .padding(adjustablePadding(for: dynamicTypeSize,
+                                       defaultButtonPadding: defaultButtonPadding))
+    }
+
+    private func adjustablePadding(for size: DynamicTypeSize, defaultButtonPadding: CGFloat) -> CGFloat {
+        switch size {
+        case .xxxLarge:       return defaultButtonPadding * 0.7
+        case .xxLarge:        return defaultButtonPadding * 0.8
+        case .xLarge:         return defaultButtonPadding * 0.9
```

Comment:
This spacing's a bit odd... you'd still have the multipliers aligned if you put each `return` on a new line

Reply:
True that, updated on https://github.com/woocommerce/woocommerce-ios/pull/14919/commits/0cddff1d6bdc2d045c957efd2cca25b7c279638e